.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,51 @@
+<!--
+Allo' allo'! 
+Thanks for taking your time to submit an issue.
+Please keep in mind, that Github issues are meant to be used for reporting bugs and to
+request new features.
+
+Use Stack Overflow for support: http://stackoverflow.com/questions/tagged/yeoman
+Head over to one of our Gitter rooms (https://gitter.im/yeoman/home) and ask for help if you’re unsure if you ran into a bug or if you have any other question.
```

Comment:
Does this makes sense?
Cause the links are within a comment block and therefore markdown doesn’t get rendered.